### ISSUE TITLE
Request splite igzip as standalone library

### ISSUE THREAD

**Issue by caoli5288** [01org/isa-l#49]
This is the fastest lib-compatible library in the world, and it shouldn't be under one directory.

**Comment by gbtucker** [comment]
I agree that igzip is special and has a wide applicability but there is some good use to keeping these 5 units together I think.  Both static and dynamic linkers are efficient in memory usage if you only use compression.  There is some sharing such as with igzip and crc.  Also a lot of adoption of functions with more than one unit.
